sycl/test/abi/pi_nativecpu_symbol_check.dump

```diff
@@ -3,184 +3,4172 @@
 # DO NOT EDIT IT MANUALLY. Refer to sycl/doc/developer/ABIPolicyGuide.md for more info.
 ################################################################################
 
-# RUN: env LLVM_BIN_PATH=%llvm_build_bin_dir %python %sycl_tools_src_dir/abi_check.py --mode check_symbols --reference %s %sycl_libs_dir/libpi_native_cpu.so
```

Comment:
My understanding is that this file should only contain the changes for the modified PI functions, but it looks like it is being replaced with the symbols exported by libsycl, is this intended? (this also applies to the other symbol check files in this PR)